src/uct/api/uct.h

```diff
@@ -519,6 +519,10 @@ enum uct_flush_flags {
                                                  error state, and it becomes
                                                  unusable for send operations
                                                  and should be destroyed. */
+    UCT_FLUSH_FLAG_REMOTE   = UCS_BIT(1)    /**< Guarantees that the target
+                                                 buffer is updated. Effective
+                                                 for RDMA write operations
```

Comment:
1. What about atomic write?
2. The API flag should be documented in UCT terms (not RDMA)
3. Will it be supported on all transports? if not, is there cap flag to query?
 For example, UCT_FLUSH_FLAG_CANCEL requires PEER_FAILURE capability.

Reply:
> * What about atomic write?

added note about atomics

> * The API flag should be documented in UCT terms (not RDMA)

rephrased, RDMA write here is just example to better explanation of flag purpose

> * Will it be supported on all transports? if not, is there cap flag to query?
>   For example, UCT_FLUSH_FLAG_CANCEL requires PEER_FAILURE capability.

this flag make any effect when remote write operation was scheduled on EP, else it is just ignored (nothing to remote complete). using this logic flag should be ignore for transports where remote completion is not actual, no need for additional capabilities

Reply:
will it be implemented for TCP and put_zcopy?

Reply:
TCP may just ignore this flag because transport itself guarantee delivery, not sure about put_zcopy - is it SW emulation over TCP or RDMA?